src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -241,21 +242,34 @@ default String embedFooterIcon() {
     }
 
     @ConfigItem(
-        keyName = "ignoredNames",
-        name = "Ignored RSNs",
-        description = "Prevent notifications from the following player names (One name per line)",
+        keyName = "ignoredNames", // historical name, preserved for backwards compatibility
+        name = "Filtered RSNs",
+        description = "Restrict what player names can trigger notifications (One name per line)<br/>" +
+            "This acts as an allowlist or denylist based on the 'RSN Filter Mode' setting below.",
         position = 1007,
         section = advancedSection
     )
-    default String ignoredNames() {
+    default String filteredNames() {
         return "";
     }
 
+    @ConfigItem(
+        keyName = "nameFilterMode",
+        name = "RSN Filter Mode",
+        description = "Allow Mode: Only allow notifications for RSNs on the list above.<br/>" +
```

Comment:
```suggestion
        description = "Allow Mode: Only allow notifications for RSNs on the list above (discouraged).<br/>" +
```

Do we want to add a warning here?